reckoner/chart.py

```diff
@@ -37,10 +37,10 @@ class Chart(object):
 
     Arguments:
     - chart (dict):
+    - helm: Instance of HelmClient()
```

Comment:
OOC, why pass an instance instead of creating one or using a singleton?

Reply:
By passing one in you can use it as a ducktype interface so that testing the class in isolation can be done easier with mocks, without lots of monkeypatching. I can show you an example in slack if you want to talk through it more. This may not be pythonic but I'd love to see any references to do this more cleanly without having to instantiate mutliple classes to test one method in one class.

Reply:
That makes sense. I honestly don't know what the most pythonic way to address this is.